### ISSUE TITLE
[Content styleguide] Content, entry points, and link labels don't use plain language or familiar words, terms, and acronyms that users understand. (05.01.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#31446]
#### VFS team name
Authenticated Experience

#### VFS product name
Notification Preferences

#### Point of Contact/Reviewers
Josh Kim (Accessibility)

---

### Platform Issue
Content, entry points, and link labels don't use plain language or familiar words, terms, and acronyms that users understand.

### Issue Details
The profile navigation uses the `aria-label` "secondary," which users may not clearly understand.

### Link, screenshot or steps to recreate
1. Log in to staging using test user 202
2. Using a screen reader, navigate by landmarks to "Secondary navigation"

### VA.gov Experience Standard
[Category Number 05, Issue Number 01](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.1.4 AAA

---

### Platform Recommendation
Instead of using an `aria-label` to label this navigation, point to the `h4` (should be a `h2`) "Profile" instead with `aria-labelledby`.

For example:
1. Add a unique-id to the `h2` "Profile" e.g. `<h2 id="unique-id">Profile</h2>`
2. Delete the attribute `aria-label="secondary"` from the nav
3. Add the attribute `aria-labelledby="unique-id"` to the nav

Ensure that this replicates on mobile as well which has the heading "Profile menu"

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by tmitchellgcio** [comment]
@joshkimux When this issue is deployed it's ready for validation. I just merged it so it may not be deployed yet

**Comment by joshkimux** [comment]
Wonderful! Thank you @tmitchellgcio 
https://user-images.githubusercontent.com/14154792/144264140-3bda1e48-24e8-4f91-b77a-2f9e7e47aafc.mov

Small note/question, we may be able to remove the `strong` element since the mobile menu now uses a `h2`

<img width="417" alt="Screen Shot 2021-12-01 at 10 36 21 AM" src="https://user-images.githubusercontent.com/14154792/144264420-99f3a3d8-22f6-4594-aef3-6fa4abfde34d.png">

**Comment by joshkimux** [comment]
Another note, which may need assistance from a designer:

The heading `profile` on desktop doesn't provide that much context that this is a list of links to navigate to different parts of the user's profile. The heading `profile` on menu _visibly_ provides more context with "menu" appended next to it, but that won't be available for screen reader users since it's outside of the `h2`.

We may want to consider:
- Nesting "menu" within the `h2` on mobile
- Nesting "menu" within a `sr-only` class on desktop

That way regardless on desktop or mobile, screen reader users will know this heading is specific to a section based on navigation (similar to an on this page heading).

**Comment by MarciMcGuireGCIO** [comment]
May need design work

**Comment by MarciMcGuireGCIO** [comment]
Will create a design task as a blocker for this.

**Comment by MarciMcGuireGCIO** [comment]
Removing blocker per [Liz's evaluation here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/33752#issuecomment-988947987).... 

Please just make the FE change exactly as Josh recommended [in his comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/31446#issuecomment-983767663).  Thanks!

**Comment by adamwhitlock1** [comment]
Yes it's ready for review! @joshkimux would you take a look at the profile menu in desktop and mobile? Thanks

**Comment by MarciMcGuireGCIO** [comment]
🥳 Thanks, @adamwhitlock1 !

**Comment by joshkimux** [comment]
<img width="1017" alt="Screen Shot 2022-01-20 at 12 58 36 PM" src="https://user-images.githubusercontent.com/14154792/150395232-2d4be614-4eae-4365-be0a-6d4afa427890.png">

Wonderful work @adamwhitlock1 👏 !!
Closing this out 🥳

**Comment by MarciMcGuireGCIO** [comment]
Thanks @adamwhitlock1 and @joshkimux !
